managed/overview.mdx

```diff
@@ -3,42 +3,38 @@ title: Overview
 description: Learn about the managed offerings we're building
 ---
 
-At Flipt, we're pioneering the next generation of feature flag management with two innovative offerings: [Bring Your Own Cloud (BYOC)](/managed/byoc) and [Hybrid Cloud](/managed/hybrid).
+At Flipt, we're pioneering the next generation of feature flag management with a new innovative offering: [Flipt Hybrid Cloud](/managed/hybrid).
 
-Both solutions are designed to seamlessly integrate into your existing infrastructure, providing unparalleled efficiency, security, and ease of use.
+This solution is designed to seamlessly integrate into your existing infrastructure, providing unparalleled efficiency, security, and ease of use.
 
-Read on to learn more about these exciting new offerings and how you can get involved.
+Read on to learn more about this exciting new offering and how you can get involved.
 
-<CardGroup>
-  <Card href="/managed/byoc" title="Bring Your Own Cloud" icon="cloud">
-    Embrace the convenience of a managed service while retaining full control in
-    your cloud environment.
-  </Card>
-  <Card href="/managed/hybrid" title="Hybrid Cloud" icon="server">
-    Combine the flexibility of deployment in your cloud with the robustness of
-    managed storage.
-  </Card>
-</CardGroup>
-
-**Bring Your Own Cloud: Customized Control in Your Cloud Environment**
-
-- Our BYOC offering ensures rapid, hassle-free deployment, expert management, and tailored integration, all within your existing cloud account.
+<Card href="/managed/hybrid" title="Hybrid Cloud" icon="server">
+  Combine the flexibility of deployment in your cloud with the robustness of
+  managed authentication and authorization.
+</Card>
 
 **Hybrid Cloud: The Best of Both Worlds**
 
 - Our Hybrid Cloud solution offers a streamlined setup, ready-to-use management interface, and simplified user and integration management, ensuring a smooth transition and enhanced operational efficiency.
 
+- Flipt Hybrid cloud handles user authentication and authorization, ensuring that your feature flags are secure and accessible only to the right people.
```

Comment:
The authz part is making sure that users are routed to their correct instance?

Reply:
I think initially, but the goal would be to offer more than just instance level authz. Since we can apply policy decisions at the HTTP layer before forwarding.